copy-file-to-s3/action.yml

```diff
@@ -70,17 +83,29 @@ runs:
           --bucket '${{ inputs.s3-bucket-name }}' \
           --key '${{ inputs.s3-bucket-path }}/${{ inputs.s3-bucket-filename }}' \
           2> /dev/null || not_exist=true
-        if [[ "${not_exist}" ]]; then
-          target_s3_filepath='${{ inputs.s3-bucket-path }}/${{ inputs.s3-bucket-filename }}'
-          if [[ '${{ inputs.compression}}' == 'true' ]]; then
-            tar -cvzf '${{ inputs.s3-bucket-filename }}' -C '${{ inputs.source-path }}' \
-              '${{ inputs.source-filename }}'
-            aws s3 cp '${{ inputs.s3-bucket-filename }}' \
-              s3://${{ inputs.s3-bucket-name }}/"${target_s3_filepath}"
-          else
-            aws s3 cp '${{ inputs.source-path }}' \
-              s3://${{ inputs.s3-bucket-name }}/"${target_s3_filepath}"
+
+        if [[ "${not_exist}" != 'true' ]]; then
+          echo '! File ${{ inputs.s3-bucket-filename }} already exists on S3 bucket.'
+
+          if [[ '${{ inputs.if-exist }}' == 'fail' ]]; then
```

Comment:
Nitpick: I'd name this argument `fallback`, as `fail` resembles an action output that is an error, and here nothing wrong happened - we just retracted with what we planned

Reply:
Changed.